README.md

```diff
@@ -7,13 +7,12 @@ transactions and querying account balances.
 
 ## Introduction
```

Comment:
I'm not sure if this is out of scope but it would be beneficial to add documentation about how this is done - like which elements of espresso is responsible for this.

Reply:
Thats a good idea, I will add that!

Reply:
https://github.com/EspressoSystems/sequencer-example-l2/pull/71/commits/9c85050f35e37c3dde45993b36089da11edafab4